### PR TITLE
Pwease and Enquote modules

### PR DESCRIPTION
**Pull request by OmarElhosseni**
Not entirely sure if I did this right, just wanted to make sure to upload it by today

### CODE REVIEW

**Comment by ErikBoesen** [comment]
You've got some merge conflicts. Also, I would recommend using git rather than editing the files on the GitHub web interface.

File: Yalebot-master/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Yalebot
```
Comment (ErikBoesen) [inline Yalebot-master/README.md]: Um... you've got a copy of the entire project in here.

File: modules/pwease.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from .base import Module
+	
+class PWEASE(Module):
```
Comment (ErikBoesen) [inline modules/pwease.py]: Don't capitalize the entire thing, just the first letter

File: modules/pwease.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from .base import Module
+	
+class PWEASE(Module):
+	def responseMaker(message):
```
Comment (ErikBoesen) [inline modules/pwease.py]: I don't see a reason to have this as a separate function. Also, use underscore_case in Python.

File: modules/pwease.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+from .base import Module
+	
+class PWEASE(Module):
+	def responseMaker(message):
+		newQuery = ""
+		for c in message.lower():
```
Comment (ErikBoesen) [inline modules/pwease.py]: You could also just do `query.replace("r", "w") + " uwu."`

File: modules/enquote.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+from .base import Module
+	
+class ENQUOTE(Module):
+    DESCRIPTION = "writes quote en quote however many times"
+    ARGC = 1
+    def response(self, query, message):
+	if(query < 0 or query > 10):
```
Comment (ErikBoesen) [inline modules/enquote.py]: Sorry but I... don't really get what this module is supposed to be for?

Comment (OmarElhosseni) [reply modules/enquote.py]: yeah, it's a pretty dumb meme that I have, doesnt really make sense to add it Im gonna delete it

**Pull request closed without merging**